libs/email-service/src/main/java/uk/gov/di/ipv/core/library/service/EmailService.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.di.ipv.core.library.service;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.library.config.ConfigurationVariable;
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class EmailService {
+
+    private static final Logger LOGGER = LogManager.getLogger();
+    private static final int NUMBER_OF_RETRIES = 3;
+    private static final int RETRY_WAIT_MILLISECONDS = 3000;
```

Comment:
Could we make these config params (in SSM) ?

Reply:
Happy to do so, but are they ever likely to change? We already have a lot of parameters in SSM and I was thinking that we could maybe avoid adding more if they weren't going to change.

Reply:
Thinking we might want to be able to easily tweak them and test them with different values in lower envs?